rpc/src/main_rpc_server.cpp

```diff
@@ -46,49 +49,53 @@ void signalHandler(int signal) {
 class RpcListener : public UListener {
 
     public:
-        UStatus onReceive(const UUri& uri,
-                          const UPayload& payload,
-                          const UAttributes& attributes) const override {
-            
+       
+         UStatus onReceive(UMessage &message) const override {
             /* Construct response payload with the current time */
             auto currentTime = std::chrono::system_clock::now();
             auto duration = currentTime.time_since_epoch();
             uint64_t currentTimeMilli = std::chrono::duration_cast<std::chrono::milliseconds>(duration).count();
 
             UPayload responsePayload(reinterpret_cast<const uint8_t*>(&currentTimeMilli), sizeof(currentTimeMilli), UPayloadType::VALUE);
-
+           
+            auto builder = UAttributesBuilder::response(message.attributes().source(), message.attributes().sink(), UPriority::UPRIORITY_CS0, message.attributes().id());
```

Comment:
this is a bug, need to use the same UUID for response

Reply:
I don't see that change in this review. It is possible that it was an artifact of this being based on other PRs, which would have been visible if this was reviewed prior to today's rebase without using the links provided in the description above.